src/theory/quantifiers/ceg_instantiator.cpp

```diff
@@ -11,7 +11,7 @@
  **
  ** \brief Implementation of counterexample-guided quantifier instantiation
  **/
- 
+
```

Comment:
Would it be possible to commit the white space changes separately (pull request not necessary, just commit to a separate branch, wait for Travis to complete and then merge to master)?

Reply:
If it's just a single line remove, I'm okay with committing this together with the rest. I'd suggest the procedure @4tXJ7f posted for larger chunks of code formatting / white space removal, but for a single line this seems a little excessive to me.

Reply:
Let me add, that I still think it's excessive if it's just introducing several single lines. But maybe we should discuss this tomorrow.

Reply:
For the record: We discussed the whitespace issue and came to the conclusion that some whitespace changes in a PR are ok.